llama_stack/providers/remote/telemetry/opentelemetry/opentelemetry.py

```diff
@@ -68,7 +68,7 @@ def __init__(self, config: OpenTelemetryConfig, deps) -> None:
 
         provider = TracerProvider(resource=resource)
         trace.set_tracer_provider(provider)
-        if TelemetrySink.JAEGER in self.config.sinks:
+        if TelemetrySink.OTEL in self.config.sinks:
```

Comment:
This file has actually been moved to https://github.com/meta-llama/llama-stack/blob/main/llama_stack/providers/inline/telemetry/meta_reference/telemetry.py. Can this change be made there?